sklearn/datasets/samples_generator.py

```diff
@@ -333,7 +334,7 @@ def make_multilabel_classification(n_samples=100, n_features=20, n_classes=5,
     generator = check_random_state(random_state)
     p_c = generator.rand(n_classes)
     p_c /= p_c.sum()
-    cumulative_p_c = np.cumsum(p_c)
+    cumulative_p_c = stable_cumsum(p_c)
```

Comment:
I don't think this is much value. `p_c` will always be high precision, and problems of numerical instability in cumulative summing aren't likely to be issue at the scale of "number of classes". Please apply this change with more discretion. It comes at a (small) cost.